include/wil/com.h

```diff
@@ -19,10 +19,10 @@
 #include "win32_helpers.h"
 #include "resource.h" // last to ensure _COMBASEAPI_H_ protected definitions are available
 
-#if __has_include(<tuple>)
+#if WI_HAS_INCLUDE(<tuple>, 1) // Tuple is C++11... assume available

```

Comment:
Maybe its just me, but the fallback being `true` (if that is possible in the preprocessor) would be more understandable.  I thought `1` meant a version number until I came across the definition of the macro.

Reply:
It seems that the C++ preprocessor _does_ understand `true` correctly. _However_ I'm a little cautious to use that here as C does not natively understand `true`. So if `true` is not defined to anything (e.g. if `stdbool.h` is not included), then `true` will get evaluated as the value `0`. In this case, that's probably a moot point - this header can't be included by C code - so it's probably fine to use `true` here. I'm just wary by default of using `true`/`false` when interfacing with the preprocessor.